README.md

```diff
@@ -28,3 +28,127 @@ After matching the slide filenames against the sample IDs from the clinical data
 
 *Note: region extraction depends on the [OpenSlide](http://openslide.org/) library. This library can be installed locally, and is also available in the Docker image described below.*
+
+# Executables for training and testing models
+
+The Docker container provides executables for training SCNN/GSCNN models and for evaluating the accuracy of these models. Both of these executables consume:
+* ROI .png files
+* A .csv file describing the patient IDs, event times, censoring (0 for uncensored, 1 for right-censored), patient indexes (integer indexes assigned to each patient), and any genomic variables. **_Note that the columns for genomic data should be organized in a way that the first coloumns are allocated for those which will NOT get normalized and the latter ones allocated for those that have to get normnalized._** The default normalization method in our model is Z score normalization. An example file is provided in this repository.
+
+## Training
+
+The command
+```
+python model_train.py
+```
+will train a model using the default hyperparameters and data locations described below.
+
+The input parameters and their default values can be listed using
+```
+python model_train.py -h
+```
+that generates output
+```
+usage: model_train.py [-h] [-m M] [-p P] [-f F] [-i I] [-r R] [-t T] [-d D]
+                      [--lr LR] [--me ME] [--kp KP] [--bs BS] [--ic IC]
+                      [--ngf NGF] [--gn GN] [--ns NS] [--nm NM]
+
+Arguments for Training the SCNN/GSCNN model
+
+optional arguments:
+  -h, --help  show this help message and exit
+  -m M        image_only or image_genome; image_only for SCNN and image_genome
+              for GSCNN. The default is image_only.
+  -p P        Path to the Train/Test patient list. The default path is ./inputs/ImageSplits.txt
+  -f F        Path to the features data. The default path is ./inputs/all_dataset.csv
+  -i I        Path to the folder containing ROI .pngs. The default path is ../images/train
+  -r R        Path where the training outputs should be generated. The default path is ./train_results
+  -t T        Path where training meta files should be generated. The default path is ./checkpoints
+  -d D        Path where binary files for training should be generated (these can be quite large and
+              should be cleaned). The default path is ./tmp
+  --lr LR     Initial learning rate. Default value = 0.001.
+  --me ME     Max number of epochs. Default value = 100.
+  --kp KP     Keeping probability for Test time dropout. Default value = 0.95.
+  --bs BS     batch size. Default value = 14.
+  --ic IC     indexes column in the number in the ./inputs/all_dataset.csv
+              file, starting from 0. Default value = 1.
+  --ngf NGF   Number of genomic features. The default is 2, considering the
+              case we have just "codeletion" and "idh mutation" as genomic
+              features.
+  --gn GN     Z Normalization of the geomic values; y for yes and n for no.
```

Comment:
I think it's ok to just mention we use z-score in the paper, and leave it to people to normalize their own features. It's much easier, and they may want to use a different method.

Reply:
I fixed it inside the code and eliminated the normalization arguments.